engine/src/cmd/ai/aggressive.cpp

```diff
@@ -173,7 +148,7 @@ static AIEvents::ElemAttrMap *getLogicOrInterrupt(string name,
         vsUMap<string, AIEvents::ElemAttrMap *> &mymap,
         int personalityseed) {
     string append = "agg";
-    static vsUMap<string, string> myappend = getAITypes();
+    static vsUMap<string, string> myappend;
```

Comment:
how will this get populated now?

Reply:
If you look at line 113-115, this is what's happening anyway. The file 'VegaPersonalities' doesn't exist in either VS or WCPU. I've trimmed the code but nothing's changed. At least, that was the intention.
I specifically asked @Loki1950, as he's our resident historian for such things. 
As I can't figure out from the code what they wanted to do and this appears to be a stub for some future feature, I vote to delete it. If/When we get to developing, we'll hopefully document it as well :)

Reply:
I just checked. The file 'VegaPersonalities.csv' does, in fact, exist in PWCU. Three copies of it. One under the 'ai' folder, another under the 'ai.easy' folder, and the third under the 'ai.hard' folder.

Reply:
Huh. I actually checked in what I assumed was the repository. Is it something you're running on your computer or is it in the repo as well? If it is, can you point me to it?

Reply:
https://github.com/pwcu/privateer_wcu/blob/master/ai.easy/VegaPersonalities.csv